autotest/gdrivers/wmts.py

```diff
@@ -41,31 +41,20 @@
 # Find WMTS driver
 
 
-def test_wmts_1():
+pytestmark = [
+    pytest.mark.require_driver('WMTS', 'WMS'),
+    pytest.mark.config_options(
+        CPL_CURL_ENABLE_VSIMEM='YES',
+        GDAL_DEFAULT_WMS_CACHE_PATH='/vsimem/cache',
+    )
+]
 
-    gdaltest.wmts_drv = gdal.GetDriverByName('WMTS')
-
-    if gdaltest.wmts_drv is not None and gdal.GetDriverByName('WMS') is None:
```

Comment:
this logic was confusing, but:
* test is skipped if the WMTS driver is not present (below)
* test is skipped if the WMTS driver *is* present but the WMTS one isn't (here)

... i.e. it just requires both drivers, which is what I've done in the mark